chart/values.yaml

```diff
@@ -1,24 +1,172 @@
+global:
+  namespace: &global-namespace gotk-system
```

Comment:
the namespace shouldn't be set as a passed value. It should be configurable on helm install via `helm install --namespace my-namespace`. This populates the global `{{ .Release.Namespace }}`